data/geolocation-cn

```diff
@@ -320,12 +321,14 @@ linkvans.com
 linuxidc.com
 liuxiaoer.com
 lu.com
+ludashi.com
```

Comment:
why not add it to `qihoo360`

Reply:
It's my mistake, I'm going to modify it now.